CAN-D/Inc/can.h

```diff
@@ -21,10 +21,19 @@
 #define CAN_RXQ_TIMEOUT_MS ((uint32_t) 250) /* Timeout when waiting on the can RX queue*/
 #define CAN_IT_START       CAN_IT_RX_FIFO0_FULL | CAN_IT_RX_FIFO1_FULL | CAN_IT_ERROR  /* Interrupts to be enabled on CAN start */
 
+/* CAN application configuration */

```

Comment:
As discussed offline, we can move this to a "bridge" file to manage configurations and data flow.